Python/chapter03/p01_three_in_one/miguelHx.py

```diff
@@ -0,0 +1,420 @@
+"""
+Python version 3.7.0
```

Comment:
FYI: Python multi-line docstrings don't  have a newline after the opening `"""` per https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings (but do have a newline before the closing `"""`, so L5 is fine).

Reply:
Updated